packages/provider/package.json

```diff
@@ -28,7 +28,7 @@
     "@metamask/safe-event-emitter": "^2.0.0",
     "@metamask/snap-types": "^0.21.0",
     "@metamask/utils": "^3.0.1",
-    "nanoid": "^4.0.0"
+    "nanoid": "^3.1.31"
```

Comment:
Why?

Reply:
`4.0.0` is ESM-only. We use `3.1.31` elsewhere.